src/plugins/intel_gpu/src/graph/impls/ocl/unique.cpp

```diff
@@ -20,6 +20,7 @@ struct unique_count_impl : typed_primitive_impl_ocl<unique_count> {
 
     std::unique_ptr<primitive_impl> clone() const override {
         return make_unique<unique_count_impl>(*this);
```

Comment:
This line should be removed

Reply:
@vladimir-paramuzov Addressed with commit 04cf3ac.